src/index.ts

```diff
@@ -97,15 +97,14 @@ app.get('/api/nonce', async (req, res) => {
 });
 
 app.get('/api/me', async (req, res) => {
-    if (!req.session.siwe) {
+    if (!req.ssx.verified) {
```

Comment:
We use the `req.ssx.verified` flag, because it validates the SSX session on each request, including fields within the SIWE message like expiration and not-before.